src/wizard/boards.ts

```diff
@@ -1,22 +1,26 @@
 import { html } from "lit";
 import { SupportedBoards } from "../api/boards";
 
-export const boardSelectOptions = (boards: SupportedBoards | undefined) =>
+export const boardSelectOptions = (
+  boards: SupportedBoards | undefined,
+  defaultBoard: string | null = null
+) =>
   boards &&
```

Comment:
this is weird. Let's just not call this function if we have no boards.

Reply:
That's how it was before my changes. I've made it wait for boards instead of checking `this._busy` now.